### ISSUE TITLE
bash `syntax error near unexpected token`

### ISSUE THREAD

**Issue by flex1988** [microsoft/inshellisense#15]
**Describe the bug**
after inshellisense bind，run bash and get error `syntax error near unexpected token`

![image](https://github.com/microsoft/inshellisense/assets/7221964/114f4862-0123-41bf-a487-4145c6882d5d)

**Comment by David-Alfred** [comment]
facing the same issue

**Comment by pstachula-dev** [comment]
Same problem on zsh

**Comment by abusaidm** [comment]
Managed to solve the issue by changing the Line Sequence in the file to LF. 

Steps:
1. Open editor of choice, I used VSCode
2. Load the target file in VSCode `~/.inshellisense/key-bindings.bash` 
3. Command pallet `Change End of Line Sequence` 
4. Save file
5. reload terminal or reload file on active terminal using `source`

![image](https://github.com/microsoft/inshellisense/assets/10223047/327f56a4-b50e-47be-b2ad-cb570b02cad4)
